stackdriver/README.md

```diff
@@ -0,0 +1,61 @@
+# stackdriver
+
+This gem is a convenience package for loading all Stackdriver gems in the google-cloud-ruby project. 
+- [google-cloud-logging](../google-cloud-logging)
+- [google-cloud-error_reporting](../google-cloud-error_reporting)
+- [google-cloud-monitoring](../google-cloud-monitoring)
+
+Please see the top-level project [README](../README.md) for more information about the individual Stackdriver google-cloud-ruby gems.
+
+## Quick Start
+
+```sh
+$ gem install stackdriver
+```
+
+## Overview
```

Comment:
We expect that the 80% use case will be for people to include only the stackdriver gem directly, and not even know/care about the other gems. So I think knowledge of the "before" case will be much less important. We can include the contrast, but if we do, it should be done "below the fold". The overview/summary docs should concentrate on how to use this gem, period.